src/main/java/seedu/address/logic/commands/AddEventCommand.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INTERVAL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REPEATABLE_DATE;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.EventList;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.repeatable.Event;
+
+public class AddEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "addEto";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the event list of the project identified "
```

Comment:
This doesn't display very nicely when printed in UI, let's add some newlines and whitespace.

Reply:
resolved